osu.Game/Localisation/GeneralSettingsStrings.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class GeneralSettingsStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.GeneralSettings";
+
+        /// <summary>
+        /// "General"
+        /// </summary>
+        public static LocalisableString GeneralSectionHeader => new TranslatableString(getKey(@"general_section_header"), @"General");
+
+        /// <summary>
+        /// "Language"
+        /// </summary>
+        public static LocalisableString LanguageHeader => new TranslatableString(getKey(@"language_header"), @"Language");
+
+        /// <summary>
+        /// "Language"
+        /// </summary>
+        public static LocalisableString LanguageDropdown => new TranslatableString(getKey(@"language_dropdown"), @"Language");
+
+        /// <summary>
+        /// "Prefer metadata in original language"
+        /// </summary>
+        public static LocalisableString PreferOriginal => new TranslatableString(getKey(@"prefer_original"), @"Prefer metadata in original language");
```

Comment:
PreferOriginalMetadataLanguage

I'm not sure why you've shortened these to the point of losing relevance. Can you do a pass and add back context?

Reply:
oh, I shortened these name is that I thought loooong variable name may decrease readability. Since you say so, I'll lookup those shortened var name and re-match the string.